src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -40,7 +40,7 @@
           (Maximum of 200 characters, no blank, /, :, - or ; characters)
         %br
         = text_field_tag(:target_package, first_template, class: 'name_appliance_form', required: true)
-        = submit_tag('Create appliance', class: 'create_appliance')
+        = submit_tag('Create appliance', class: 'create_appliance', disabled: User.current.is_nobody? ? true: nil)
```

Comment:
It is necessary because for ``disable`` it doesn't matter the value. So even for ``disable: false`` the button would be disabled. So we need to set ``nil`` here to not set the attribute at all!